src/components/AutoCompleteCustomer.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { useGetCustomersQuery } from '@/redux/services/propertiesApi';
+
+const AutoCompleteCustomer = ({ onSelect }) => {
```

Comment:
Maybe this autcomplete could be a component generic and we could receive just the data and the handle event as parameter.